### ISSUE TITLE
Test this repository for KITTI Dataset(Not 360)

### ISSUE THREAD

**Issue by Lee-JaeWon** [Miaosheng1/nerf_kitti360#2]
Thank you for your work.

Would it be possible to use the same code for KITTI (Not 360)?

Or is there something else we need to consider?

**Comment by Miaosheng1** [comment]
No, I have not tested this code on KITTI datasets. Anyway, for 3D Neural Reconstruction task, it is important to get accurate camera extrinsic and intrinsic parameters. this means if you can read the accurate KITTI camera parameters, I think you can apply this code for KITTI.
